test/test_managercli.py

```diff
@@ -574,6 +574,16 @@ def test_unset(self):
     def test_unset_and_add_and_remove(self):
         self._test_exception(['--add', 'test', '--remove', 'item', '--unset'])
 
+    def test_valid_value(self):
+        with patch.object(self.cc, '_get_valid_fields') as mock_get_valid_fields:
+            mock_get_valid_fields.return_value = {'addons': ['ADDON1', 'ADDON3', 'ADDON2']}
+            self.assertEqual(self.cc._are_provided_values_valid(['ADDON1']), [])
+
+    def test_invalid_value(self):
+        with patch.object(self.cc, '_get_valid_fields') as mock_get_valid_fields:
+            mock_get_valid_fields.return_value = {'addons': ['ADDON1', 'ADDON3', 'ADDON2']}
+            self.assertEqual(self.cc._are_provided_values_valid(['test']), ['test'])
```

Comment:
You can also capture output of `_are_provided_values_valid` and test if output is expected. Something like this:

```python
        with Capture() as cap:
            result = self.cc._are_provided_values_valid(['FOO'])
        self.assertTrue('Some warning message...' in cap.out)
```